packages/peregrine/lib/hooks/useQuery.js

```diff
@@ -27,7 +27,7 @@ export const useQuery = query => {
      * @param {DocumentNode} query A GraphQL document
      */
     const runQuery = useCallback(
-        async ({ variables }) => {
+        async ({ variables } = {}) => {
```

Comment:
This is a QOL fix. Right now if you call `runQuery` for a query that doesn't accept variables you have to pass an empty object. This makes that unnecessary so you can just call `runQuery()` instead of `runQuery({})`.